cookbook/email/dev_environment.rst

```diff
@@ -119,6 +119,57 @@ the replaced address, so you can still see who it would have been sent to.
     These are ``X-Swift-Cc`` and ``X-Swift-Bcc`` for the ``CC`` and ``BCC``
     addresses respectively.
 
+Sending to a Specified Address, but with exceptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Suppose you normally in your dev environment want to have all email redirected to a specific address,
+(like in the above scenario to ``dev@example,com``).
+But then you may want email sent to some specific email addresses to go through after all,
+and not be redirected (even if it is in the dev environment).
+This can be done by adding the ``delivery_whitelist`` option:
```

Comment:
could you please reformat this to only add line breaks after the first word that crosses the 72th character? I.e:

````rst
Suppose you normally in your dev environment want to have all email redirected
to a specific address, (like in the above scenario to ``dev@example,com``).
But then you may want email sent to some specific email addresses to go through
after all, and not be redirected (even if it is in the dev environment).  This
can be done by adding the ``delivery_whitelist`` option:
````